docs/releases/v2022.1.rst

```diff
@@ -0,0 +1,416 @@
+
+-  Linksys
+
+   -  WRT160NL
+
+-  TP-Link
+
+   -  TL-MR13U (v1)
+   -  TL-MR3020 (v1)
+   -  TL-MR3040 (v1, v2)
+   -  TL-MR3220 (v1, v2)
+   -  TL-MR3420 (v1, v2)
+   -  TL-WA701N/ND (v1, v2)
+   -  TL-WA730RE (v1)
+   -  TL-WA750RE (v1)
+   -  TL-WA801N/ND (v1, v2, v3)
+   -  TL-WA830RE (v1, v2)
+   -  TL-WA850RE (v1)
+   -  TL-WA860RE (v1)
+   -  TL-WA901N/ND (v1, v2, v3, v4, v5)
+   -  TL-WA7210N (v2)
+   -  TL-WA7510N (v1)
+   -  TL-WR703N (v1)
+   -  TL-WR710N (v1, v2)
+   -  TL-WR740N (v1, v3, v4, v5)
+   -  TL-WR741N/ND (v1, v2, v4, v5)
+   -  TL-WR743N/ND (v1, v2)
+   -  TL-WR840N (v2)
+   -  TL-WR841N/ND (v3, v5, v7, v8, v9, v10, v11, v12)
+   -  TL-WR841N/ND (v1, v2)
+   -  TL-WR843N/ND (v1)
+   -  TL-WR940N (v1, v2, v3, v4, v5, v6)
+   -  TL-WR941ND (v2, v3, v4, v5, v6)
+   -  TL-WR1043N/ND (v1)
+   -  WDR4900
+
+-  Ubiquiti
+
+   -  AirGateway
+   -  AirGateway Pro
+   -  AirRouter
+   -  Bullet
+   -  LS-SR71
+   -  Nanostation XM
+   -  Nanostation Loco XM
+   -  Picostation
+
+-  Unknown
+
+   -  A5-V11
+
+-  VoCore
+
+   -  VoCore (8M, 16M)
+
+Atheros target migration
+------------------------
+
+All Atheros MIPS devices built with the ``ar71xx-generic``,
+``ar71xx-nand`` as well as ``ar71xx-tiny`` were deprecated upstream and
+are therefore not available with Gluon anymore.
+
+Many devices previously built with ``ar71xx-generic`` and
+``ar71xx-nand`` are now available with the ``ath79-generic`` as well as
+``ath79-nand`` target respectively.
+
+Missing devices
+~~~~~~~~~~~~~~~
+
+The following devices have not yet been integrated into Gluons ath79
+targets.
+
+-  8Devices
+
+   -  Carambola 2
+
+-  Aerohive
+
+   -  HiveAP 121
+
+-  Allnet
+
+   -  ALL0315
+
+-  Buffalo
+
+   -  WZR-HP-G300NH2
+   -  WZR-HP-G450H
+
+-  GL.iNet
+
+   -  6408A v1
+
+-  NETGEAR
+
+   -  WNDR4300
+   -  WNDRMAC
+   -  WNDRMAC v2
+
+-  TP-Link
+
+   -  WR2543
+
+- Ubiquiti
+
+   -  Rocket
+
+-  WD
+
+   -  MyNet N600
+   -  MyNet N750
+
+-  ZyXEL
+
+   -  NB6616
+   -  NB6716
+
+Features
+--------
+
+Wireguard
+~~~~~~~~~
+
+Gluon got WireGuard support. This allows offloading **encrypted**
+connections into kernel space, increasing performance by forwarding
+packets without the need for context switches between user and kernel
+space.
+
+In order to reuse existing (already verified) fastd-keypairs for
+WireGuard, a key derivation procedure is `currently being
+developed <https://github.com/freifunk-gluon/gluon/pull/2601>`__. This
+should ease migration from fastd to WireGuard in case whitelisting VPN
+keys is desired.
+
+fastd L2TP
+~~~~~~~~~~
+
+fastd can now act as a connection broker for unencrypted L2TP-based
+tunneling within Gluons mesh-vpn framework. This new ``null@l2tp``
+connection Method allows for increased performance within existing
+fastd setups.
+
+In addition to a sufficiently
+:ref:`configured fastd-based VPN server<vpn-gateway-configuration>`,
+this requires further modifications to a site :ref:`VPN fastd methods`.
```

Comment:
```suggestion
this requires further modifications to a sites :ref:`VPN fastd methods<VPN fastd methods>`.
```

rendered as `[...] modifications to a site Method.`
now renders: `[...] modifications to a sites fastd methods.`